armi/bookkeeping/report/data.py

```diff
@@ -193,6 +192,10 @@ def __init__(self, title, description="", header=None):
 
     def __str__(self):
         """Truer to content representation."""
+        # error handling
+        if not len(self.data):
+            return ""
```

Comment:
This is not, strictly speaking, code coverage. I found this edge case error in the reporting code, while trying to write a unit test.

So, testing works!